opennlp-tools/src/main/java/opennlp/tools/ml/model/DataIndexer.java

```diff
@@ -80,9 +81,14 @@
   /**
    * Sets parameters used during the data indexing.
    * @param trainParams
-   * @param reportMap
    */
-  public void init(Map<String, String> trainParams, Map<String, String> reportMap);
+  public void init(Map<String, String> trainParams);
```

Comment:
Didn't we just add this method after 1.7.0 release with your big PR? In that case I would remove it again, otherwise it would have to be deprecated first.